gammapy/conftest.py

```diff
@@ -12,6 +12,10 @@
 ## from the list of packages for which version numbers are displayed
 ## when running the tests ... this was added in Astropy 1.0
 try:
+    # Removing h5py for now to get rid of this travis-ci fail
+    # https://travis-ci.org/gammapy/gammapy/jobs/86565582#L1074
+    del PYTEST_HEADER_MODULES['h5py']
```

Comment:
@astrofrog @mwcraig Since today I'm getting this error from travis-ci:
https://travis-ci.org/gammapy/gammapy/jobs/86570163#L1061
```
INTERNALERROR>   File "h5py/h5t.pxd", line 14, in init h5py._conv (/home/ilan/minonda/conda-bld/work/h5py/_conv.c:7364)
INTERNALERROR>   File "h5py/numpy.pxd", line 66, in init h5py.h5t (/home/ilan/minonda/conda-bld/work/h5py/h5t.c:20460)
INTERNALERROR> ValueError: numpy.dtype has the wrong size, try recompiling
```

I tried `del PYTEST_HEADER_MODULES['h5py']` in `gammapy/conftest.py`, but am still getting the error, i.e. pytest is still trying to import `h5py` to get it's version number.

Have you seen this issue with Astropy or other affiliated packages?
Any idea how to resolve it?